src/app/modules/shared-components/components/add-content/add-content.component.ts

```diff
@@ -29,7 +29,9 @@ export class AddContentComponent<Type> implements OnInit, OnDestroy {
   @Input() allowedTypesForNewContent: NewContentType<Type>[] = [];
   @Input() searchFunction?: (searchValue: string) => Observable<AddedContent<Type>[]>;
   @Input() loading = false;
+  @Input() addedItemIds: string[] = [];
```

Comment:
Add content component is not item specific !

Reply:
Fixed